packages/ndla-ui/src/i18n/__tests__/formatMessage-test.ts

```diff
@@ -1,34 +0,0 @@
-/**
```

Comment:
Måtte slette denne for at det skulle kjøre. Ser ut som det er noe som henger igjen fra i18n omskrivning.

Reply:
Hva slags feil får du? Hos meg får jeg både buildet, kjørt og testet. Dette er forsåvidt litt etterslep fra i18n-omskriving. Funksjonaliteten brukes fortsatt i article-converter, så det er fint å ha tester på det.

Reply:
Aner ikke hvorfor det ikke kjørte i går, men det ser ut som om det ikke hadde noe med denne å gjøre. Det fungerer hos meg og nå. La den tilbake igjen